### ISSUE TITLE
Feature Request: Gesture Indicators

### ISSUE THREAD

**Issue by thereallinusrg** [bubelov/news#69]
Preamble:
After happily using the app for over half a year, I had to confess to myself that I had never red the pop up appearing after performing a swipe gesture (mark read or bookmark).

I found that I bookmarked hundereds of articles that I wanted to mark as red.

Request:
Add a color indicator for the swipe gestures. E.g. green for mark red and yellow for bookmark. 
This UI element would prevent pop up ignoring users like me to performe unwanted actions.

Some Email apps like for example ProtonMail for Android use color indicators for swipe gestures.

**Comment by bubelov** [comment]
Related: https://github.com/bubelov/news/issues/20

**Comment by bubelov** [comment]
@thereallinusrg changing background color wouldn't work here, but adding icons might help. I tried a few approaches and here is what I ended up with. It will be rolled out with the next release.

![Screenshot_20210528-223143_News](https://user-images.githubusercontent.com/1189245/120009353-1172b900-c006-11eb-9bc7-dc7d8ed4caf9.jpg)
![Screenshot_20210528-222721_News](https://user-images.githubusercontent.com/1189245/120009357-133c7c80-c006-11eb-92f4-2cf956b93e9d.jpg)
